jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskListPresenterTest.java

```diff
@@ -51,4 +54,11 @@ public void testResumeActionCondition() {
         testTaskStatusCondition(getPresenter().getResumeActionCondition(),
                                 TASK_STATUS_SUSPENDED);
     }
+    
+    @Test
+    public void testReleaseActionOwnerCondition() {
```

Comment:
Like in the previous commit: how about renaming the method to `public void userShouldNotBeAbleToReleaseTasksOwnedByOthers`